bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/localizedfields.js

```diff
@@ -178,11 +178,6 @@ pimcore.object.classes.data.localizedfields = Class.create(pimcore.object.classe
                     fieldLabel: t("label_width"),
                     value: this.datax.labelWidth
                 },
-                {
-                    xtype: "displayfield",
-                    hideLabel: true,
-                    value: t('width_explanation')
```

Comment:
Can the associated translation be removed? Or is it still used somewhere else?

Reply:
Is still used in the width/height fields, where % values are possible